utils/spdx/src/main/kotlin/SpdxExpression.kt

```diff
@@ -107,10 +107,9 @@ sealed class SpdxExpression {
         }
 
     /**
-     * Normalize all license IDs using a mapping containing common misspellings of license IDs. If [mapDeprecated] is
-     * `true`, also deprecated IDs are mapped to their current counterparts. The result of this function is not
-     * guaranteed to contain only valid IDs. Use [validate] to check the returned [SpdxExpression] for validity
-     * afterwards.
+     * Normalize all license IDs using [SpdxSimpleLicenseMapping]. If [mapDeprecated] is `true`, also deprecated IDs are
+     * mapped to their current counterparts. The result of this function is not guaranteed to contain only valid IDs.
+     * Use [validate] to check the returned [SpdxExpression] for validity afterwards.
```

Comment:
out of scope: Should this be `isValid()` instead of `validate()`, because the latter does not return any value?

Reply:
I believe the mention of `validate()` better matches the `normalize()` context, but I've now simply mentioned `isValid()` as well.